src/main/webapp/jsp/instructorCourseEnrollResult.jsp

```diff
@@ -1,125 +1,82 @@
-    <script type="text/javascript" src="//ajax.googleapis.com/ajax/libs/jqueryui/1.10.4/jquery-ui.min.js"></script>
-    <script type="text/javascript" src="/js/common.js"></script>
-    <script type="text/javascript" src="/bootstrap/js/bootstrap.min.js"></script>
-
-    <jsp:include page="../enableJS.jsp"></jsp:include>
-
-    <script type="text/javascript" src="/js/date.js"></script>
+<c:set var="jsIncludes">
     <script type="text/javascript" src="/js/instructor.js"></script>
-</head>
+</c:set>
 
-<body>
-    <jsp:include page="<%=Const.ViewURIs.INSTRUCTOR_HEADER%>" />
+<c:set var="COURSE_ID">
+    <%=Const.ParamsNames.COURSE_ID%>
+</c:set>
 
-    <div class="container" id="mainContent">
-        <div id="topOfPage"></div>
-        <h1>Enrollment Results for <%=sanitizeForHtml(data.courseId)%></h1>
-        <br>
+<c:set var="STUDENTS_ENROLLMENT_INFO">
+    <%=Const.ParamsNames.STUDENTS_ENROLLMENT_INFO%>
+</c:set>
 
-        <div class="alert alert-success">
-        <form name='goBack' action="<%=data.getInstructorCourseEnrollLink(data.courseId)%>" method="post" role="form"> 
+<ti:instructorPage pageTitle="TEAMMATES - Instructor" bodyTitle="Enrollment Results for ${data.courseId}" jsIncludes="${jsIncludes}">
+    <div class="alert alert-success">
+        <form name='goBack' action="${data.instructorCourseEnrollLink}" method="post" role="form"> 
             Enrollment Successful. Summary given below. Click <a id="edit_enroll" href="javascript:document.forms['goBack'].submit()">here</a> to do further changes to the student list.
-            
-        <input type="hidden" name="<%=Const.ParamsNames.COURSE_ID%>" value="<%=data.courseId%>">
-        <input type="hidden" name="<%=Const.ParamsNames.STUDENTS_ENROLLMENT_INFO%>" value="<%=data.enrollStudents%>">
+            <input type="hidden" name="${COURSE_ID}" value="${data.courseId}">
+            <input type="hidden" name="${STUDENTS_ENROLLMENT_INFO}" value="${data.enrollStudents}">
         </form>
-        </div>
-        
-        <%
-            for(int i=0; i < 6; i++){
-                List<StudentAttributes> students = data.students[i];
-        %>
-            <%
-                if(students.size()>0){
-            %>  
-                <% if(i == 0){ %>
-                <div class="panel panel-danger">
-                <% } else if(i == 1){ %>
-                <div class="panel panel-primary">
-                <% } else if(i == 2){ %>
-                <div class="panel panel-warning">
-                <% } else if(i == 3){ %>
-                <div class="panel panel-info">
-                <% } else if(i == 4){ %>
-                <div class="panel panel-default">
-                <% } else{ %>
-                <div class="panel panel-danger">
-                <% } %>
+    </div>
+    
+    <c:forEach items="${data.students}" var="studentList" varStatus="i">
+        <c:if test="${not empty studentList}">
+            <c:choose>
```

Comment:
This segment is pretty hard to understand.
Would it be a good idea to create a data structure + tag for it?
The data structure could store maps linking the enum to both the class (danger, primary, warning, etc) and the enrollment status message.